routes/payments.js

```diff
@@ -2,25 +2,42 @@ require('dotenv').config();
 const { STRIPE_PRIVATE_API_KEY = "missing_key" } = process.env;
 const stripe = require('stripe')(STRIPE_PRIVATE_API_KEY);
 const paymentsRouter = require("express").Router();
+const { isLoggedIn } = require("./util")
 
-const { YOUR_DOMAIN = 'http://localhost:3000' } = process.env;
+// Do we really need the domain if this is going to route to /payments/?success=true   ?
+// const { YOUR_DOMAIN = 'http://localhost:3000' } = process.env;
```

Comment:
This is where we need to add our stripe key. You will need to make an account to receive one.